probe-rs-tools/src/bin/probe-rs/cmd/cargo_embed/rttui/tab.rs

```diff
@@ -132,7 +126,7 @@ impl Tab {
         }
 
         let old_message_count = self.messages.len();
-        match &up_channel.data {
+        match &up_channels[self.up_channel].data {
```

Comment:
Same here :)

Reply:
Same here ;) Panic is essentially impossible, but if not, the panic message is more informative and at the same time, the syntax is simpler.